src/util/vue-compat/index-2.js

```diff
@@ -17,6 +17,24 @@ export function getDefaultSlot(component) {
   return component.$slots.default;
 }
 
+export function renderToString(app) {
+  let _renderToString;
+  try {
+    _renderToString = require('vue-server-renderer/basic');
```

Comment:
why not via dynamic import, does it not work?

Reply:
I haven't tried dynamic import with the vue 2 part. It should work. 
However, its ESM output will include dynamic import syntax and we don't know if our customers' setup supports it. So technically speaking, it could break. So I left the vue 2 part as-is.

Reply:
we probably should release this in a major version, for these small changes though 🤔

Reply:
Hmmm do you see something other than this?

Reply:
it will make it clearer from which version vue 3 is supported, and we don't need to be as careful to prevent breaks where we think they make sense